src/cli/Cli.ts

```diff
@@ -137,8 +140,7 @@ export class Cli {
       showHelp ||
       parsedArgs.h ||
       parsedArgs.help) {
-      this.printHelp(this.getHelpMode(parsedArgs));
```

Comment:
I find it unclear that we open the browser from a function that's called printHelp. Better naming is hard, because it does too much. Some adherence to the single responsibility principle would be good.

Maybe we can separate it into a function that gets the help path and two separate functions that do something with that path : a printHelp and openHelpPage, or something like that. 

What do you think?

Reply:
Great idea

Reply:
Hi Martin

I've been thinking about this quite a lot and I'm really struggling with this one. It's really hard to explain, but we call the printHelp functionallity in multiple occasions:

When we specify `m365 --help`, we will call the functionality, but we will never open in the browser, as `m365` is not a command to execute, and we will log the available commands, however, this functionallity is in the same function...

I could go and refactor everything, and it would look like this: 
![image](https://github.com/pnp/cli-microsoft365/assets/9193870/afceb3c5-caec-4a9e-b304-af67a60b730b)

Which in my opinion, looks fairly decent; but I still struggle with finding a name for the method (Why is this always the hardest part of writing code... 😉)

If you have any suggestions, feel free to tell me.